### ISSUE TITLE
Thanks for this repo

### ISSUE THREAD

**Issue by josephspurrier** [ricallinson/covfmt#3]
I was able to get coveralls to show the coverage of two different languages thanks to this tool:
https://coveralls.io/builds/23731268

I've forked it so that I can add in the full path of the file since the full path is missing from the Go cover file - here is how I use it in my Travis CI file: https://github.com/josephspurrier/govueapp/blob/b530517e5448a88b8a1861bf9ce54e0b1b59661c/.travis.yml#L43

Also, I had to make a change to this line because I didn't have any underscores in my cover file:
https://github.com/josephspurrier/covfmt/blob/master/main.go#L89

**Comment by ricallinson** [comment]
Glad it help you. Do you want to put up a PR of your changes?

**Comment by ricallinson** [comment]
Thinking about it the underscore removal can become an option as it was a hack anyway.

**Comment by ricallinson** [comment]
I've kicked of a new branch to add tests and your changes; https://github.com/ricallinson/covfmt/tree/cli

**Comment by josephspurrier** [comment]
Yeah, let's get this consolidated - your tests are failing randomly because you're looping through a map in the lcov function. Can you change blocks to an array so output will be reliable? Also, can you hook it up to Travis and Coveralls - both are free and that will help with running tests on each commit.

**Comment by ricallinson** [comment]
Good catch on the map. Thanks. Travis is hooked up now and I'm looking at Coveralls (never heard of it before).
